sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -251,8 +254,8 @@ def tune(
             pip_index_url: The PyPI url from which to install Python packages.
             metrics_collector_config: Specify the config of metrics collector, 
                 for example, `metrics_collector_config = {"kind": "Push"}`.
-                Currently, we only support `StdOut` and `Push` metrics collector.
```

Comment:
I think we may need to tell users about the supported types of MC. So can you re-add this line?